scripts/ci/utils/generate-changelog

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+[ -n "${DEBUG}" ] && set -x
+
+# This script is intended to be called by the ci scripts, please dont
+# execute it directly.
+
+## Configuration
+#####################################################################
+
+# path of this file
+SCRIPT_PATH="$(cd $(dirname "$0")/$(dirname "$(readlink "$0")") && pwd)"
+
+# project root for this file
+PROJECT_ROOT="$( cd "$( echo ${SCRIPT_PATH} | sed s+/scripts/ci/utils++)" && pwd )"
+
+# If not empty, generate changelog from this tag instead of last release
+LAST_RELEASE_TAG=${LAST_RELEASE_TAG:-""}
+
+## Script
+#####################################################################
+
+# get tags to get changes between
+TAGS=($(git -C "${PROJECT_ROOT}" tag -l --sort=-version:refname))
```

Comment:
❓ Need to check whether this works with the version prefix.

Reply:
`| grep $PKG_TARGET` ✅